src/main/resources/api/service_openapi.yaml

```diff
@@ -34,8 +34,6 @@ paths:
       tags:
         - janitor
       requestBody:
-        description: |
-          Resource UID, time to live, resource labels
```

Comment:
Does taking out the `|` and making this a non-literal string work? Or do you want to get rid of this?

Reply:
Even got complaints in the old line 124 where I don't have the `|`

Reply:
Yeah, you can't have anything next to a $ref. You can use "allOf" to combine a ref and a description, though, or else just put the description inside the $ref. This line wouldn't have that problem, though.